modules/ROOT/pages/_partials/install-mule-runtime-versions.adoc

```diff
@@ -17,7 +17,36 @@ If you change the Java or Mule runtime versions in your project, consider these
 * Standard support for Java 8 and 11 ends in March 2025 for Mule 4.8 Edge and August 2025 for 4.6 LTS, so plan your upgrade path for apps that are running on Java 8 or 11 accordingly.
 * Mule runtime 4.9 LTS and Edge run and compile only on Java 17. To use Mule 4.9, upgrade your apps to run on Java 17. 
 
-See the xref:studio-release-notes.adoc[Studio release notes] for the latest information on compatibility.
+== Mule Runtime and Java Compatibility in Studio
+
+This table shows the compatibility between Mule runtime and Java versions in Studio:
+
+[cols="1,1,1,1,1",options="header"]
+|===
+|Studio Version | Mule Runtime Version | Java 8 | Java 11 | Java 17
+
+| 7.21 and later
+a| 
+* 4.9.x (LTS) and later
+* 4.8.x
+* 4.6.x (LTS)
+| No 
+| No 
+| Yes
+
+| 7.20 
+a| 
+* 4.8.x and later
+* 4.7.x
+* 4.6.x (LTS)
+* 4.5.x 
```

Comment:
Should we mention those versions out of support? 4.5 is EoL, and I understand that 4.7 went EoL with 4.9 release

Reply:
You're right! According to Mule runtime release notes, 4.5 and 4.7 are EoL. 
Great catch! Thanks!